libmachine/provision/ubuntu.go

```diff
@@ -49,11 +49,14 @@ func (provisioner *UbuntuProvisioner) Service(name string, action pkgaction.Serv
 func (provisioner *UbuntuProvisioner) Package(name string, action pkgaction.PackageAction) error {
 	var packageAction string
 
+	updateMetadata := true
```

Comment:
Ah, it's a default, I see.

Probably a little nit-picky, but do you mind moving it into a var block like so at the top of the function?

```go
var (
    packageAction   string
    updateMetadata bool
)
```